src/main/java/frc/robot/Ports.java

```diff
@@ -10,8 +10,14 @@
         public static final boolean IS_AUX_SENSOR_INVERTED = false;
     }
 
+    public static class Shooter {
+        public static final int MAIN_MOTOR = 0; // Main motor port.
+        public static final boolean IS_MAIN_INVERTED = false; // Whether the motor is inverted.
+        public static final boolean MAIN_SENSOR_PHASE = false; // Whether the encoder is inverted.
```

Comment:
If you don't know how many motors you will end up with then reflect it in the comments.